drivers/shared/executor/test-resources/busybox/README

```diff
@@ -0,0 +1,3 @@
+Downloaded busybox https://busybox.net/downloads/binaries/
+
+Busybox binaries are statically linked binaries, that is a multi-call binary.  It's a single binary that can act like many commonly used utilities (e.g. /bin/sh, echo, sleep, etc).  More info is found in https://busybox.net/downloads/BusyBox.html .
```

Comment:
I don't think we do.  It's a compiled binary GPLv2 that's unmodified.  We host busybox docker tarballs in other test paths as well.  I'll add a note about it in README just in case though.